docs/Tutorial_DiscordMERLIN.ipynb

```diff
@@ -0,0 +1,1401 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #54.            include = np.ones(n, dtype=bool)
</pre><p>This would be <code>np.ones(k, dtype=bool)</code>  <span class="ql-cursor">﻿</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
since some comments  are related, I am going to show a Thumb's Up and then close it if I don't have any question. Is that okay with you?

Reply:
Thank you!